### ISSUE TITLE
IFile FileSize (or Length)

### ISSUE THREAD

**Issue by DaleyKD** [pnp/pnpcore#422]
## Category

- [x] Domain model extension

## Describe the domain model extension
In CSOM, [Microsoft.SharePoint.Client.File](https://docs.microsoft.com/en-us/previous-versions/office/sharepoint-csom/jj167343(v=office.15)) has a property named `Length`.

### Model

Model: IFile
Properties: Length (or FileSize/Size)

### APIs
API: `https://<tenant>.sharepoint.com/sites/<site>/_api/web/lists(guid'<listId>')/items(<listItemId>)/file` _(`length` property)_
or
API: `https://graph.microsoft.com/v1.0/sites/<siteId>/lists/<listId>/items/<itemId>/driveItem` _(`size` property)_

### Additional Information
[Drive Item Microsoft Graph Documentation](https://docs.microsoft.com/en-us/graph/api/resources/driveitem?view=graph-rest-1.0)

**Comment by DaleyKD** [comment]
I'm hoping to be able to submit a PR at some point, but I'm fighting being able to get the unit tests to work.  I can't seem to `Register-PnPManagementShellAccess`.  https://github.com/pnp/powershell/issues/663

**Comment by jansenbe** [comment]
@DaleyKD :  really appreciate you wanne help out :-) If you're in GCC then please follow what was mentioned in the other PS issue. Once you've the app setup you need to update the test configuration files to use it...that would then be something like this:

```json
"Credentials": {
      "DefaultConfiguration": "MySetup",
      "Configurations": {
        "MySetup": {
          "CredentialManager": {
            "ClientId" : "<your app id>",
            "TenantId" : "<your tenant id>",
            "CredentialManagerName": "credman"
          }
        }
      }
    },
```

Alternatively you can also sign up for a free developer tenant via https://developer.microsoft.com/en-us/microsoft-365/dev-program and then you're tenant admin in a regular tenant, so the documented flow should work.

If in the end you're not able to get things working then please reach out, we can have a short meeting to get you unblocked.

**Comment by DaleyKD** [comment]
Tried to fix this with #445

**Comment by jansenbe** [comment]
Thanks a ton @DaleyKD , just merged your PR so closing this issue. Thanks for helping out the SharePoint community.
